Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -152,6 +150,50 @@ private static void ShowPlayerResult(Dictionary<string, int> playerResult)
                 WriteColoredMessage($" { playerResults.Key} -> {playerResults.Value}", color);
             }
         }
+        private static void StartGame()
+        {
+            int gameNumber = 1;
+            string repeatGame = "y";
+
+            while(repeatGame == "y")
+            {
+                ShowGameNumber(gameNumber);
+
+                Dictionary<string, int> playerResult = StartRiddle();
+
+                ShowPlayerResult(playerResult);
+
+                SeperateGames();
+
+                repeatGame = AskRepeat();
+               
+                gameNumber++;
+            }
+        }
+
+        private static void ShowGameNumber(int gameNumber)
+        {
+            Console.WriteLine();
+            Console.WriteLine($"================================= GAME #{gameNumber} =================================");
+            Console.WriteLine();
+        }
+
+        private static void SeperateGames()
```

Comment:
Introduce new method `WriteLine` which would accept 2 boolean params (add new line before message & add new line after message) and use this new method in all places where you need to write message in new line (Console.WriteLine)